services/wallet/thirdparty/utils/symbols.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import "strings"
+
+var renameMapping = map[string]string{
+	"STT": "SNT",
+}
+
+func RenameSymbols(symbols []string) (renames []string) {
+	for _, symbol := range symbols {
+		renames = append(renames, GetRealSymbol(symbol))
+	}
+	return
+}
+
+func GetRealSymbol(symbol string) string {
+	if val, ok := renameMapping[strings.ToUpper(symbol)]; ok {
+		return val
+	}
+	return strings.ToUpper(symbol)
+}
+
+func ChunkSymbols(symbols []string) [][]string {
+	var chunks [][]string
+	chunkSize := 20
```

Comment:
Does this chunk size come from a provider limitation or to limit how much data we download in one go? Anyway, it's something we probably want to define somewhere else and pass to this method as a parameter